microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -78,12 +70,13 @@ You can now configure audit logging policies to manage the retention policies fo
 ==== Multiple networks capability now available
 With this release, using multiple networks is supported with the {microshift-short} Multus plugin. If you have advanced networking requirements, you can attach additional networks to a pod for high-performance network configurations. After installing the {microshift-short} Multus RPM package, you can use the Bridge, MACVLAN, or IPVLAN plugins to create additional networks. See xref:../microshift_networking/microshift_multiple_networks/microshift-cni-multus.adoc#microshift-multus-intro_microshift-cni-multus[Additional networks in {microshift-short}].
 
+[id="microshift-4-16-configure-ingress-router"]
+==== Custom configurations for the ingress router are supported
+You can now configure ingress routes to completely isolate your devices, or to maintain separate network connections for them. You can use a variety of combinations to customize the endpoint configuration for your use case. See xref:../microshift_networking/microshift-nw-router.adoc#microshift-nw-router-con_microshift-nw-router[About configuring the router].
```

Comment:
I think that it is inaccurate to say that ingress routes are used "to completely isolate your devices, or to maintain separate network connections for them". I suggest removing this sentence.

Reply:
Updated to a more general benefit. WDYT?